cmd/trade.go

```diff
@@ -538,6 +538,7 @@ func runTradeCmd(options inputs) {
 		env,
 		runtime.GOOS,
 		runtime.GOARCH,
+		runtime.Version(),
```

Comment:
I think this has been passed in for the `goarm` param instead of the `go_version` param